client/request/consts.go

```diff
@@ -20,6 +20,7 @@ const (
 	DocKey      = "dockey"
 	DocKeys     = "dockeys"
 	FieldName   = "field"
+	FieldIDName = "fieldId"
```

Comment:
Note: new property required, as some references to `FieldName` were actually for `field` in stuff like aggregates.

Reply:
There was already `FieldIDFieldName` bellow.

Reply:
`FieldIDFieldName` is the field name, not the input arg name.